xbmc/windowing/windows/WinSystemWin32.cpp

```diff
@@ -444,6 +444,9 @@ bool CWinSystemWin32::ChangeResolution(RESOLUTION_INFO res)
 {
   const MONITOR_DETAILS &details = GetMonitor(res.iScreen);
 
+  if (!&details)
```

Comment:
on win32 it did what it should, but I had a bad feeling, I agree that it should return a pointer.